hummedia/resources.py

```diff
@@ -103,6 +108,42 @@ def set_query(self):
         else:
             q["oauth"][self.manual_request["provider"]]=self.manual_request["provider_id"]
         return q
+
+    def get_enrollments(self, username, userid):
+        from flask import request, Response, jsonify, current_app, session
+        from flask_oauth import OAuth
+        from datetime import datetime, timedelta, date
+        from functools import update_wrapper
+        from mongokit import cursor
+        from bson import ObjectId
+        from models import connection
+        from config import APIHOST, YT_SERVICE, BYU_WS_ID, BYU_SHARED_SECRET
+        from urllib2 import Request, urlopen, URLError
+        import json, byu_ws_sdk, requests, re, os, mimetypes
+        import time
+        import xml.etree.ElementTree as ET
+
+        def getCurrentSem():

```

Comment:
Is this function always accurate?  Doesn't spring/summer share the month of June or something?